docs/icons/2021_Devfile_logo_Config_A.svg

```diff
@@ -0,0 +1 @@
,0,1,4.39-20.35,33.25,33.25,0,0,1,12.33-13.95,32.6,32.6,0,0,1,17.71-5q15.54,0,24,9.91t8.47,28v6.89H892.09q.76,9.4,6.29,14.9a18.88,18.88,0,0,0,13.88,5.48,23.11,23.11,0,0,0,19.11-9.49l9.21,8.79a30.86,30.86,0,0,1-12.19,10.58A38.3,38.3,0,0,1,911.27,599.46Zm-2-65.16a14.4,14.4,0,0,0-11.35,4.92q-4.32,4.92-5.51,13.71h32.54v-1.27c-.38-5.71-1.9-10-4.57-13S914,534.3,909.23,534.3Z"/><path class="cls-1" d="M979,576.4,995.05,522h17.64l-26.35,76H971.51l-26.57-76h17.71Z"/><path class="cls-1" d="M1029.35,598.05v-63.4h-11.6V522h11.6v-7q0-12.66,7-19.54t19.68-6.89a39.53,39.53,0,0,1,9.55,1.26l-.42,13.36a34,34,0,0,0-6.53-.56q-12.24,0-12.23,12.58V522h15.46v12.65h-15.46v63.4Z"/><path class="cls-1" d="M1077.28,502.25a9,9,0,0,1,2.5-6.53c1.66-1.74,4-2.6,7.13-2.6s5.48.86,7.17,2.6a9,9,0,0,1,2.53,6.53,8.77,8.77,0,0,1-2.53,6.43c-1.69,1.72-4.08,2.57-7.17,2.57s-5.47-.85-7.13-2.57A8.82,8.82,0,0,1,1077.28,502.25Zm18.13,95.8h-17.07V522h17.07Z"/><path class="cls-1" d="M1129.29,598.05h-17.08v-108h17.08Z"/><path class="cls-1" d="M1184.25,599.46q-16.23,0-26.32-10.23T1147.85,562v-2.1a45.61,45.61,0,0,1,4.39-20.35,33.25,33.25,0,0,1,12.33-13.95,32.6,32.6,0,0,1,17.71-5q15.54,0,24,9.91t8.47,28v6.89h-49.69q.77,9.4,6.29,14.9a18.88,18.88,0,0,0,13.88,5.48,23.11,23.11,0,0,0,19.11-9.49l9.21,8.79a30.86,30.86,0,0,1-12.19,10.58A38.3,38.3,0,0,1,1184.25,599.46Zm-2-65.16a14.4,14.4,0,0,0-11.35,4.92q-4.32,4.92-5.51,13.71h32.54v-1.27c-.38-5.71-1.9-10-4.57-13S1186.94,534.3,1182.21,534.3Z"/><path class="cls-1" d="M1238.65,580.06q4.78,0,7.38,2.7a9.42,9.42,0,0,1,2.6,6.79,9.08,9.08,0,0,1-2.6,6.64q-2.6,2.63-7.38,2.63a9.9,9.9,0,0,1-7.24-2.6,8.89,8.89,0,0,1-2.67-6.67,9.42,9.42,0,0,1,2.6-6.79A9.67,9.67,0,0,1,1238.65,580.06Z"/><path class="cls-1" d="M1271.12,502.25a9.06,9.06,0,0,1,2.49-6.53c1.67-1.74,4-2.6,7.13-2.6s5.49.86,7.17,2.6a9,9,0,0,1,2.53,6.53,8.77,8.77,0,0,1-2.53,6.43c-1.68,1.72-4.07,2.57-7.17,2.57s-5.46-.85-7.13-2.57A8.86,8.86,0,0,1,1271.12,502.25Zm18.13,95.8h-17.08V522h17.08Z"/><path class="cls-1" d="M1304.64,559.32a44.83,44.83,0,0,1,4.43-20.13,32.32,32.32,0,0,1,12.44-13.78,35.13,35.13,0,0,1,18.41-4.81q15.39,0,25,9.91t10.37,26.28l.07,4a45.24,45.24,0,0,1-4.33,20.1,32,32,0,0,1-12.37,13.71,35.32,35.32,0,0,1-18.59,4.85q-16.08,0-25.76-10.72t-9.66-28.57Zm17.08,1.48q0,11.73,4.85,18.38a15.75,15.75,0,0,0,13.49,6.64,15.57,15.57,0,0,0,13.46-6.75q4.82-6.74,4.82-19.75,0-11.52-5-18.27a16.7,16.7,0,0,0-26.74-.11Q1321.72,547.59,1321.72,560.8Z"/><path class="cls-2" d="M693.73,550A39,39,0,0,0,679,519.44a11.4,11.4,0,0,1-3.12,5.83l-3,3a28.32,28.32,0,1,1-18.77-6.69l-9.1,9.1a5.4,5.4,0,0,0,7.63,7.63l14-14h0l4.14-4.14a4.25,4.25,0,0,0,.27-5.7c-.12-.05-.81-.83-1.1-1.12L652.62,496a5.4,5.4,0,0,0-7.63,7.64l7.25,7.25A39.13,39.13,0,1,0,693.73,550Z"/><path class="cls-2" d="M732.52,551A79.36,79.36,0,1,0,652,630.33l-7,7a5.4,5.4,0,0,0,7.63,7.63l16.2-16.2h0l2-2a4.27,4.27,0,0,0,0-6L668,617.94h0l-15.3-15.3a5.4,5.4,0,0,0-7.63,7.64l9.25,9.25-1.08,0a68.64,68.64,0,1,1,22.7-3.89l0,.05A11.43,11.43,0,0,1,679,626,79.47,79.47,0,0,0,732.52,551Z"/></svg>
```

Comment:
```suggestion
<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1920 1080"><defs><style>.cls-1{fill:#494949;}.cls-2{fill:#2f9af2;}</style></defs><path class="cls-1" d="M788.27,598.05V495.72h30.22a47.36,47.36,0,0,1,24.07,6,41.13,41.13,0,0,1,16.27,17.15,54.32,54.32,0,0,1,5.77,25.44v5.13q0,14.55-5.8,25.59a40.75,40.75,0,0,1-16.52,17,49.6,49.6,0,0,1-24.56,6Zm17.78-88v73.79h11.6q14,0,21.47-8.75T846.75,550v-5.7q0-16.65-7.24-25.44t-21-8.78Z"/><path class="cls-1" d="M911.27,599.46q-16.23,0-26.32-10.23T874.87,562v-2.1a45.61,45.61,0,0,1,4.39-20.35,33.25,33.25,0,0,1,12.33-13.95,32.6,32.6,0,0,1,17.71-5q15.54,0,24,9.91t8.47,28v6.89H892.09q.76,9.4,6.29,14.9a18.88,18.88,0,0,0,13.88,5.48,23.11,23.11,0,0,0,19.11-9.49l9.21,8.79a30.86,30.86,0,0,1-12.19,10.58A38.3,38.3,0,0,1,911.27,599.46Zm-2-65.16a14.4,14.4,0,0,0-11.35,4.92q-4.32,4.92-5.51,13.71h32.54v-1.27c-.38-5.71-1.9-10-4.57-13S914,534.3,909.23,534.3Z"/><path class="cls-1" d="M979,576.4,995.05,522h17.64l-26.35,76H971.51l-26.57-76h17.71Z"/><path class="cls-1" d="M1029.35,598.05v-63.4h-11.6V522h11.6v-7q0-12.66,7-19.54t19.68-6.89a39.53,39.53,0,0,1,9.55,1.26l-.42,13.36a34,34,0,0,0-6.53-.56q-12.24,0-12.23,12.58V522h15.46v12.65h-15.46v63.4Z"/><path class="cls-1" d="M1077.28,502.25a9,9,0,0,1,2.5-6.53c1.66-1.74,4-2.6,7.13-2.6s5.48.86,7.17,2.6a9,9,0,0,1,2.53,6.53,8.77,8.77,0,0,1-2.53,6.43c-1.69,1.72-4.08,2.57-7.17,2.57s-5.47-.85-7.13-2.57A8.82,8.82,0,0,1,1077.28,502.25Zm18.13,95.8h-17.07V522h17.07Z"/><path class="cls-1" d="M1129.29,598.05h-17.08v-108h17.08Z"/><path class="cls-1" d="M1184.25,599.46q-16.23,0-26.32-10.23T1147.85,562v-2.1a45.61,45.61,0,0,1,4.39-20.35,33.25,33.25,0,0,1,12.33-13.95,32.6,32.6,0,0,1,17.71-5q15.54,0,24,9.91t8.47,28v6.89h-49.69q.77,9.4,6.29,14.9a18.88,18.88,0,0,0,13.88,5.48,23.11,23.11,0,0,0,19.11-9.49l9.21,8.79a30.86,30.86,0,0,1-12.19,10.58A38.3,38.3,0,0,1,1184.25,599.46Zm-2-65.16a14.4,14.4,0,0,0-11.35,4.92q-4.32,4.92-5.51,13.71h32.54v-1.27c-.38-5.71-1.9-10-4.57-13S1186.94,534.3,1182.21,534.3Z"/><path class="cls-1" d="M1238.65,580.06q4.78,0,7.38,2.7a9.42,9.42,0,0,1,2.6,6.79,9.08,9.08,0,0,1-2.6,6.64q-2.6,2.63-7.38,2.63a9.9,9.9,0,0,1-7.24-2.6,8.89,8.89,0,0,1-2.67-6.67,9.42,9.42,0,0,1,2.6-6.79A9.67,9.67,0,0,1,1238.65,580.06Z"/><path class="cls-1" d="M1271.12,502.25a9.06,9.06,0,0,1,2.49-6.53c1.67-1.74,4-2.6,7.13-2.6s5.49.86,7.17,2.6a9,9,0,0,1,2.53,6.53,8.77,8.77,0,0,1-2.53,6.43c-1.68,1.72-4.07,2.57-7.17,2.57s-5.46-.85-7.13-2.57A8.86,8.86,0,0,1,1271.12,502.25Zm18.13,95.8h-17.08V522h17.08Z"/><path class="cls-1" d="M1304.64,559.32a44.83,44.83,0,0,1,4.43-20.13,32.32,32.32,0,0,1,12.44-13.78,35.13,35.13,0,0,1,18.41-4.81q15.39,0,25,9.91t10.37,26.28l.07,4a45.24,45.24,0,0,1-4.33,20.1,32,32,0,0,1-12.37,13.71,35.32,35.32,0,0,1-18.59,4.85q-16.08,0-25.76-10.72t-9.66-28.57Zm17.08,1.48q0,11.73,4.85,18.38a15.75,15.75,0,0,0,13.49,6.64,15.57,15.57,0,0,0,13.46-6.75q4.82-6.74,4.82-19.75,0-11.52-5-18.27a16.7,16.7,0,0,0-26.74-.11Q1321.72,547.59,1321.72,560.8Z"/><path class="cls-2" d="M693.73,550A39,39,0,0,0,679,519.44a11.4,11.4,0,0,1-3.12,5.83l-3,3a28.32,28.32,0,1,1-18.77-6.69l-9.1,9.1a5.4,5.4,0,0,0,7.63,7.63l14-14h0l4.14-4.14a4.25,4.25,0,0,0,.27-5.7c-.12-.05-.81-.83-1.1-1.12L652.62,496a5.4,5.4,0,0,0-7.63,7.64l7.25,7.25A39.13,39.13,0,1,0,693.73,550Z"/><path class="cls-2" d="M732.52,551A79.36,79.36,0,1,0,652,630.33l-7,7a5.4,5.4,0,0,0,7.63,7.63l16.2-16.2h0l2-2a4.27,4.27,0,0,0,0-6L668,617.94h0l-15.3-15.3a5.4,5.4,0,0,0-7.63,7.64l9.25,9.25-1.08,0a68.64,68.64,0,1,1,22.7-3.89l0,.05A11.43,11.43,0,0,1,679,626,79.47,79.47,0,0,0,732.52,551Z"/></svg>

```
here and below, but that's not requirement

Reply:
These svg files are created by the design folks so I'd like to keep them unmodified unless it is needed.